partner_firstname/models/res_partner.py

```diff
@@ -89,11 +91,25 @@ def default_get(self, fields_list):
         return result
 
     @api.model
+    def _get_names_order(self):
+        """Get names order configuration from system parameters.
+        You can override this method to read configuration from language,
+        country, company or other"""
+        return self.env['ir.config_parameter'].get_param(
+            'partner_names_order', 'last_first')
```

Comment:
I think it's no necessary because there's a default value, but I'm going to encapsulate it in a method in order to be able to inherit from it easly